ngraph/frontend/generic/include/frontend_manager/frontend_manager.hpp

```diff
@@ -115,9 +115,15 @@ class NGRAPH_API Place
     /// For operation node returns reference to an input port with specified index
     virtual Ptr getInputPort (int inputPortIndex = -1) const;
 
+    /// For operation node returns reference to an input port with specified name and index
+    virtual Ptr getInputPort (const std::string& inputName, int inputPortIndex = -1) const;
```

Comment:
BTW, -1 was a temporary stub for a new method without inputPortIndex. Similarly for other methods. @nosovmik, @itikhono, do you think reasonable to split each of such methods to a pair of method with and w/o port index? Or even remove methods that have default port index requiring to set it to 0 even if there is only a single port in operation. nGraph follows this approach.

Reply:
I personally prefer to avoid usage of virtual functions with default arguments :-)

Reply:
actually default arguments in virtual functions do nothing, inhereted classes do not use it